plugins/embark-assistant/finder_ui.cpp

```diff
@@ -419,6 +553,56 @@ namespace embark_assist {
 
                 break;
 
+                case fields::spire_count_min:
+                case fields::spire_count_max:
+                    for (int16_t k = -1; k <= 9; k++) {
+                        if (k == -1) {
+                            element->list.push_back({ "N/A", k });
+                        }
+                        else {
+                            element->list.push_back({ std::to_string(k), k });
+                        }
+                    }
+
+                    break;
+
+                case fields::magma_min:
+                case fields::magma_max:
+                {
+                    embark_assist::defs::magma_ranges k = embark_assist::defs::magma_ranges::NA;
+                    while (true) {
+                        switch (k) {
+                        case embark_assist::defs::magma_ranges::NA:
+                            element->list.push_back({ "N/A", static_cast<int8_t>(k) });
+                            break;
+
+                        case embark_assist::defs::magma_ranges::Cavern_3:
+                            element->list.push_back({ "3:rd Cavern", static_cast<int8_t>(k) });
```

Comment:
Unsure if these correspond to `1`, `2`, `3` keys - if they don't, I would get rid of the colons.

Reply:
That't the bug report. I'll add a reference to it.

I have trouble understanding the comment. There are lots of colons in there, and no explicit key press references. However after reading it several times my guess is that you refer to 1:st, 2:nd, and 3:rd? Do you think it would read better as First, Second, and Third?

Reply:
Yeah, I meant "3:rd", preferring "3rd" or "Third".